maistra/ci/pre-submit.sh

```diff
@@ -18,7 +18,58 @@ export ARCH
 # Fix path to the vendor deps
 sed -i "s|=/work/|=$(pwd)/|" maistra/bazelrc-vendor
 
-# Build
+# Build WASM extensions first
+bazel build \
+  --incompatible_linkopts_to_linklibs \
```

Comment:
`incompatible_linkopts_to_linklibs` is used to correct the order of libraries on the command line under software collections environment (devtools-9 in our case). Don't need this with clang, unless it's being installed from a software collection.

Reply:
Ditto for other places where it's used here.

Reply:
We are building with gcc. We are using clang only to build the WASM extensions, which are not affected by this flags (they are a separate toolchain).